src/components/get-involved/mobilepopupCard.jsx

```diff
@@ -0,0 +1,63 @@
+/* add and import styles */
+/* view and test it out by importing it to the respective page */
+import { useTranslations } from 'next-intl';
+import React, { useState } from "react";
+import { IoIosCloseCircleOutline } from "react-icons/io";
+import styles from '@/styles/components/get-involved/mobilepopupCard.module.scss';
+import GetInvolved from '@/pages/get-involved';
+
+export async function getStaticProps({ locale }) {
+  return {
+    props: {
+      messages: (await import(`@/messages/${locale}.json`)).default
+    }
+  };
+}
+
+export default function PopupCard({ card }) {
+  console.log(card);
+  const t = useTranslations(`GetInvolved.PopupCard.${card}`);
+  const [showFirstContent, setShowFirstContent] = useState(true);
+  const toggleContent = () => {
+    setShowFirstContent(!showFirstContent);
+  };
+
+  return (
+    <div className={styles.multicontentcard}>
+      <div className={styles.layout}>
+        <h2>{t('title')}</h2>
```

Comment:
be sure to add a close button with the react-icon similar to how you did on the desktop version right before the title